src/components/tooltip/Marker.js

```diff
@@ -227,16 +212,26 @@ function _markerBacktrace(
         </TooltipDetails>
       );
     default:
-      break;
+      return null;
   }
-  if ('cause' in data && data.cause) {
+}
+
+function _markerBacktrace(
+  marker: Marker,
+  thread: Thread,
+  implementationFilter: ImplementationFilter
+): React.Node {
+  const { data, start } = marker;
+  if (data && 'cause' in data && data.cause) {
     const { cause } = data;
-    const causeAge = marker.start - cause.time;
+    const causeAge = start - cause.time;
     return (
       <div className="tooltipDetailsBackTrace" key="backtrace">
-        <h2 className="tooltipBackTraceTitle">
-          First invalidated {formatNumber(causeAge)}ms before the flush, at:
-        </h2>
+        {data.type === 'Styles' ? (
```

Comment:
[`Styles`](https://searchfox.org/mozilla-central/rev/06bd14ced96f25ff1dbd5352cb985fc0fa12a64e/tools/profiler/core/ProfilerMarkerPayload.cpp#283) is a special kind of marker that has its own marker payload type and additional information: 
[Checked the reflow markers](https://searchfox.org/mozilla-central/rev/06bd14ced96f25ff1dbd5352cb985fc0fa12a64e/layout/base/PresShell.cpp#9184) and yes, they also have a cause, so we can add this data to those markers as well but I would prefer doing it in another PR since this customization also was happening only in `Styles` marker data type before. I can open an issue for that and can do it after this PR. What do you think?

Reply:
Filed #2108 to track that.